### DISCUSSION TITLE
Refined backup functionality

### DISCUSSION THREAD

**Discussion by Freewindrider** [akeeba/panopticon#575] [Ideas]
Hej 

It is already easily possible to setup a backup plan for each site. 
Old backups can be deleted manually.

**How about refining the automated backup process by bringing the quota management functions of Akeeba Backup Pro to Panopticon with the opportunity to (re-)configure the number of (obsolete) records/backups to keep?** 

I guess, this would be a nice completion for the backup section.

**Comment by nikosdion** [comment]
This is controlled in Akeeba Backup itself, not Panopticon. Panopticon is simply an Akeeba Backup API client. It lists the backups you see in Akeeba Backup, and runs backups on a schedule. Everything else is always, **always** controlled in Akeeba Backup itself.

**Reply by Freewindrider** [reply]
**This makes sense,** it's no problem to change the respective settings in Akeeba Backup. 

I thought it could be useful, in case you notice "huge" sites getting close to their hdd-/ssd-capacity when storing a certain number of backups. 
You could then reduce the nr. to keep for the default backup profile directly in Panopticon.
